### PR TITLE
Update compiled version

### PR DESCRIPTION
**Pull request by mroethke**
This updates the compiled version.

Must not be merged before #3!

### CODE REVIEW

**Comment by smlng** [comment]
please rebase and rebuild after all other PRs are merged now

**Review by smlng — CHANGES_REQUESTED** [review]
see comment

File: docs/_sources/usage.rst.txt [hunk 1]
```diff
@@ -64,6 +64,42 @@ First, setup Homebrew [#homebrew]_ and afterwards install the RTRlib package:
...
+
+.. rubric:: Footnotes
+
+.. [#overlay] https://github.com/FRRouting/gentoo-overlay
+.. [#layman] https://wiki.gentoo.org/wiki/Layman
+
+
+Debian
+------
+
```
Comment (fho) [inline docs/_sources/usage.rst.txt:88]: a link to https://packages.debian.org/search?searchon=names&keywords=librtr would be handy

File: docs/_sources/usage.rst.txt [hunk 2]
```diff
@@ -64,6 +64,42 @@ First, setup Homebrew [#homebrew]_ and afterwards install the RTRlib package:
...
+Debian contains RTRlib since Buster [#DebianBuster]_ and can be installed with apt.
+The package is called ``librtr0``, developers will require the ``librtr-dev`` package, too.
+
+.. code-block:: bash
+
+    apt install librtr0 librtr-dev
```
Comment (fho) [inline docs/_sources/usage.rst.txt:94]: Like described for the gentoo part, it could be added how to add the buster repository to your apt sources list

Comment (mroethke) [reply docs/_sources/usage.rst.txt:94]: That is not comparable. For gentoo it is an additional repository compatible with mainline gentoo. For Debian you would have to include the repository of a completely different Debian version, which you absolutely do not want to do. Unless you want to upgrade.

File: docs/_sources/usage.rst.txt (hunk 1 above)
Comment (mroethke) [reply docs/_sources/usage.rst.txt:88]: Why?

**Comment by mroethke** [comment]
@smlng done

File: docs/_sources/usage.rst.txt (hunk 1 above)
Comment (fho) [reply docs/_sources/usage.rst.txt:88]: To easily be able to checkout in which apt-repositories the rtrlib is available, which rtrlib debian packages exist and in which versions

Comment (smlng) [reply docs/_sources/usage.rst.txt:88]: this updates the compiled version of the handbook, if you want that link in please provide a PR that changes/adds that in the source file (and recompile again).

**Review by smlng — APPROVED** [review]

**Pull request merged**